types/components/messages.d.ts

```diff
@@ -13,7 +13,12 @@ export interface PayPalMessagesComponentOptions {
             size?: 10 | 11 | 12 | 13 | 14 | 15 | 16;
             align?: "left" | "center" | "right";
         };
+        ratio?: "1x1" | "1x4" | "8x1" | "20x1";
     };
+    placement?: "home" | "category" | "product" | "cart" | "payment";
+    onApply?: Function;
+    onClick?: Function;
+    onRender?: Function;
```

Comment:
```suggestion
    onRender?: () => void;
```

Reply:
When testing locally I noticed that these callbacks do have data that's passed in. I don't think we need to type it all out at this point but it's good to call it out with `data: Record<string, unknown>`. Ex:
<img width="1579" alt="Screen Shot 2022-07-29 at 9 50 49 AM" src="https://user-images.githubusercontent.com/534034/181786112-974857cf-c1ac-4013-b57d-36b1b70627eb.png">

Reply:
Yea, that was my delay in accepting. I was noticing paypal-messaging-components tests seems to have some data being passed in but tracing what and where it goes was getting a little confusing.